03:27:48 test1.py:superlongâ€¦:29 ERROR Could not run function
03:27:48 test1.py:<module>:35 INFO {
    "timestamps": {"always_show": False, "use_utc": False},
    "stacking": {"enabled": True, "case_sensitive": True},
    "formatting": {
        "ansi": True,
        "highlighting": True,
        "pretty_print": True,
        "fixed_format_width": 0,
    },
    "filtering": {
        "min_level": 0,
        "exclude_messages": [],
        "include_only_messages": [],
    },
    "output": {"default_file_stream": None},
    "metadata": {
        "show_metadata": False,
        "include_timestamp": False,
        "include_level_name": False,
        "include_thread_name": True,
        "include_file_name": False,
        "include_wrapping_function": False,
        "include_function": True,
        "include_line_number": False,
        "include_value_count": True,
    },
    "log_line": {"format": "default"},
}
03:27:48 test1.py:<module>:36 INFO {"enabled": True, "case_sensitive": True}
03:27:48 test1.py:<module>:37 INFO <futils.log.handlers.StackingRules object at 0x10132bfe0>
03:27:48 test1.py:<module>:38 INFO True
03:27:48 test1.py:<module>:41 INFO a
3 4 4 hello world 3
03:27:48 test1.py:<module>:42 INFO a344hello world 3
03:27:48 test1.py:<module>:43 INFO a
3 4 4 hello world 3 ["a\n\n\n\n", 3, 4.0, 4.2, "hello world 3"]
03:27:48 test1.py:<module>:46 INFO a 3 4 4 hello world 3
03:27:48 test1.py:<module>:47 INFO a_3_4_4_hello world 3
03:27:48 test1.py:<module>:48 INFO a
03:27:48 test1.py:<module>:49 INFO ajkfhgfhkgdjhd
03:27:48 test1.py:<module>:50 INFO 345
03:27:48 test1.py:<module>:52 DEBUG 3.0
03:27:48 test1.py:<module>:53 DEBUG 3.0 {}
03:27:48 test1.py:<module>:54 DEBUG Dictionary: {}
03:27:48 test1.py:<module>:60 LETHAL {
    "abcdef": "abcdef",
    "l": ["l", ["l", ["l", ["l", ["l", ["l", []]]]]]],
    "mnopqr": "mnopqr",
    "stuvwx": "stuvwx",
    "hi": (33, <futils.log.logger.Logger object at 0x10132bf20>),
}
03:27:48 test1.py:<module>:70 ERROR hi bye
03:27:48 test1.py:<module>:71 WARNING The given tuple is not a valid RGB: (132, 204, 256)
03:27:48 test1.py:<module>:76 WARNING The given tuple is not a valid RGB: (132, 204, 256)
03:27:48 test1.py:<module>:77 WARNING The given tuple is not a valid RGB: (132, 204, 256)
03:27:48 test1.py:<module>:78 WARNING The given tuple is not a valid RGB: (132, 204, 256)
03:27:48 test1.py:<module>:84 DEBUG Added groups: ["log1.log", "log2.log", "log3.log"]
 test1.py:<module>:94 DEBUG Added groups: ["log4.log"]
